osbs/cli/main.py

```diff
@@ -719,9 +717,7 @@ def cli():
     build_type_group.add_argument("--worker", action="store_true", required=False,
                                   default=False, help="create worker build")
     build_type_group.add_argument("--orchestrator", action="store_true", required=False,
-                                  default=False, help="create orchestrator build")
-    build_type_group.add_argument("--prod", action="store_true", required=False,
```

Comment:
May dropping of this option cause any issues? But, well we noticed others about deprecation of this one early